tic_tac_toe/game.py

```diff
@@ -63,22 +86,53 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
-
+    #check for 1st diagonal
+    if _is_winning_combination(board,((0,0),(1,1),(2,2)), player):
+        return player
+    
+    #check for 2nd diagonal
+    elif _is_winning_combination(board,((0,2),(1,1),(2,0)), player):
+        return player
+    
+    #check horizontally:
+    elif _is_winning_combination(board,((0,0),(0,1),(0,2)), player):
+        return player
+    elif _is_winning_combination(board,((1,0),(1,1),(1,2)), player):
+        return player        
+    elif _is_winning_combination(board,((2,0),(2,1),(2,2)), player):
+        return player
+    
+    #check vertically
+    elif _is_winning_combination(board,((0,0),(1,0),(2,0)), player):
+        return player
+    elif _is_winning_combination(board,((0,1),(1,1),(2,1)), player):
+        return player    
+    elif _is_winning_combination(board,((0,2),(1,2),(2,2)), player):
+        return player
+    
+    return None
```

Comment:
You don't need to `return None` here, if nothing is returned it's the same thing as returning None.